prime-angular-frontend/src/app/shared/models/email-pair.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface EmailPair {
```

Comment:
Can this be named `EmailsForCareSetting` or something more descriptive?

Reply:
Good idea. Naming is updated.

Reply:
Can you also please rename the file accordingly?

Reply:
ops, file renamed.